README.md

```diff
@@ -1,90 +1,42 @@
 # Bitcoin Design Guide
 
-This is a project overview of a design guide for bitcoin with early thoughts on what it could end up being.
```

Comment:
I kinda like this story adds a bit of personality. :smile:

Reply:
travel guide story is in the design guide itself now